### PR TITLE
Add new things to do

### PR DESCRIPTION
**Pull request by mcapkovic**

### CODE REVIEW

File: things_to_do.js [hunk 1]
```diff
@@ -1,170 +1,262 @@
...
+    {
+        name: "Cook something you've never cooked before",
+        link: ''
+    },
+    {
+        name: 'Reconnect with a friend',
+        link: ''
+    },
+    {
+        name: 'Close at least 5 tabes in your browser',
```
Comment (lukas-cap) [inline things_to_do.js]: nit: tabes -> tabs

File: things_to_do.js [hunk 2]
```diff
@@ -1,170 +1,262 @@
...
+    {
+        name: 'Relish a mechanical butterfly',
+        link: 'https://youtu.be/1gu3z7w4Vc8'
+    },
+    {
+        name: 'Think about the fact that brown does not exist',
+        link: 'https://youtu.be/wh4aWZRtTwU'
+    },
+    {
+        name: 'Learn about Snake Island',
```
Comment (lukas-cap) [inline things_to_do.js]: nit: the Snake Island

File: things_to_do.js [hunk 3]
```diff
@@ -1,170 +1,262 @@
...
+    {
+        name: 'Crush your friend in Rock Paper Scissors',
+        link: 'https://www.youtube.com/watch?v=rudzYPHuewc'
+    },
+    {
+        name: 'Learn the best skill for your resume',
+        link: 'https://www.reddit.com/r/AskReddit/comments/1sn8ki/what_is_a_skill_someone_can_learn_in_6_months/cdzjm0e?utm_source=share&utm_medium=web2x'
+    },
+    {
+        name: 'Learn a paradox',
```
Comment (lukas-cap) [inline things_to_do.js]: Maybe better wording would be: "Learn about paradoxes" ?

File: things_to_do.js [hunk 4]
```diff
@@ -1,170 +1,262 @@
...
+        link: 'https://www.reddit.com/r/AskReddit/comments/4f50wo/whats_the_best_paradox_you_know/'
+    },
+    {
+        name: 'Read a short comics',
```
Comment (lukas-cap) [inline things_to_do.js]: nit: comic

File: things_to_do.js [hunk 5]
```diff
@@ -1,170 +1,262 @@
...
+    {
+        name: 'Clean your desktop',
+        link: ''
+    },
+    {
+        name: 'Clean your home screen',
+        link: ''
+    },
+    {
+        name: 'Count binary on you fingers',
```
Comment (lukas-cap) [inline things_to_do.js]: Better wording: "Learn to count in binary on your fingers" ?

File: things_to_do.js [hunk 6]
```diff
@@ -1,170 +1,262 @@
...
+        link: 'https://www.youtube.com/watch?v=apCLHmPsC68'
+    },
+    {
+        name: 'Have you ever had a dream like this?',
```
Comment (lukas-cap) [inline things_to_do.js:250]: Maybe let's get rid of this one? :D I want to keep it wholesome and this is a bit too metal.

File: things_to_do.js [hunk 7]
```diff
@@ -1,170 +1,262 @@
...
+    {
+        name: 'Learn about alternatives to google search',
+        link: 'https://fossbytes.com/google-alternative-best-search-engine/'
+    },
+    {
+        name: 'Watch Gourmet Pringles attempt',
+        link: 'https://www.youtube.com/watch?v=J3xLG-whfFk'
+    },
+    {
+        name: 'Learn how to make a diamonds carbonated water',
```
Comment (lukas-cap) [inline things_to_do.js]: Maybe better wording: "Learn how to make the most expensive carbonated water" . People won't really get if diamonds is a brand or it's actual diamonds we're talking about. If you want to keep diamonds in there maybe: "Learn how to make carbonated water with diamonds"

File: things_to_do.js [hunk 8]
```diff
@@ -1,170 +1,262 @@
...
+        link: 'https://youtu.be/n0wvDwSnzcw'
+    },
+    {
+        name: 'Relish a mechanical butterfly',
```
Comment (lukas-cap) [inline things_to_do.js]: Maybe just "Look at these mechanical butterflies" ?

**Review by lukas-cap — CHANGES_REQUESTED** [review]
Also please bump the ?v= version param for this file in index.html so people don't get the cached version.

File: things_to_do.js (hunk 6 above)
Comment (mcapkovic) [reply things_to_do.js:250]: changed for non-metal version :D

**Pull request merged**